README.md

```diff
@@ -1,5 +1,9 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/layer5-no-trim.svg" alt="Layer5-logo" width="45%" />
+   <picture align="center">
```

Comment:
Will need improvement, dark mode is ok but light mode is buggy 
![image](https://user-images.githubusercontent.com/95926324/193838913-831f859d-edb5-4192-a82c-922bbd54cc5c.png)

Reply:
That's weird. It looks un-buggy over here. 🤔 

![Screenshot_20221004-150534_Chrome.jpg](https://user-images.githubusercontent.com/105166953/193841058-3feb44d3-f524-43d1-acf8-7a41c3daacb8.jpg)

Reply:
Ah...yes.
I found the perfect image to normalise aspect ratio!

Commiting changes in 2 seconds. 🙂🚀